### DISCUSSION TITLE
useFetch - how to download file from server?

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17976] [Questions]
please help - tell me how get file from server.... file like PDF document

**Comment by wobsoriano** [comment]
Curious why you ended up with `useFetch`?

Check answers [here](https://stackoverflow.com/questions/32545632/how-can-i-download-a-file-using-window-fetch) on how to download a file using `window.fetch`

**Reply by mihail727** [reply]
I just thought maybe this can be done by nuxt
